apps/redi-connect/src/pages/app/applications/ApplicationsFilterContext.tsx

```diff
@@ -0,0 +1,99 @@
+import { createContext, ReactNode, useState } from 'react'
+import { useSelector } from 'react-redux'
+import { RedMatch } from '@talent-connect/shared-types'
+import { RootState } from '../../../redux/types'
+
+interface ApplicationsFilterContextProps {
+  activeFilter: string
```

Comment:
Great work creating this context! There's a nice low-hanging fruit & TS improvement here we can add as the topping on the coding cake. Hint: union types. Catch my drift?